examples/CalcQt/CMakeLists.txt

```diff
@@ -12,8 +12,8 @@ if(QT_LIBRARIES)
     if(Qt5TEST_FOUND)
         add_executable(QtTestCalculatorQtSteps features/step_definitions/QtTestCalculatorQtSteps)
         set_target_properties(QtTestCalculatorQtSteps PROPERTIES AUTOMOC ON)
-        target_include_directories(QtTestCalculatorQtSteps PUBLIC src)
-        target_link_libraries(QtTestCalculatorQtSteps libcalcqt ${QT_LIBRARIES} ${CUKE_LIBRARIES})
+        target_include_directories(QtTestCalculatorQtSteps PRIVATE src)
```

Comment:
Looking at this again. You're probably adding this for the `examples/CalcQt/src/CalculatorWidget.h` file. But that's already reachable because you're linking to `libcalcqt`. I.e. CMake propagates the usage requirement of `libcalcqt` to this target. You shouldn't need this entire `target_include_directories` directive.